src/components/SurveyQuestion/SurveyQuestion.jsx

```diff
@@ -51,31 +52,22 @@ const SurveyQuestion = ({
         ))}
       </ul>
       <div>
-        {!isFirstQuestion && (
-          <button
-            className="navigation-button"
-            onClick={() => {
-              onNavigate(previousRoute);
-            }}
-          >
-            👈 Previous
-          </button>
-        )}
-        <button
-          className="navigation-button"
-          onClick={() => {
+        <QuestionerButton
+          isFirstQuestion={isFirstQuestion}
+          isLastQuestion={isLastQuestion}
+          nextRoute={nextRoute}
+          previousRoute={previousRoute}
+          onNavigate={onNavigate}
+          Finish={isLastQuestion}
+          BackRoute={previousRoute}
+          startRoute={startRoute}
```

Comment:
Unused prop, let's remove it

Reply:
if remove those props the next button is not working